### DISCUSSION TITLE
How to auto download / grab every day ?

### DISCUSSION THREAD

**Discussion by Jimpan81** [iptv-org/community#448] [Q&A] [answered]
How to auto download / grab every day  on OSX ? I have the terminal way working but want to automate it 😁

**Comment by PopeyeTheSai10r** [comment, marked as answer]
Hello @Jimpan81, You wanna grab the EPG (Electronic Program Guide)?

[How to use?](https://github.com/iptv-org/epg#how-to-use)

**Reply by Jimpan81** [reply]
Thanks! Yeah that is the way i do it , but now i want to have it done automatic every day . How to do that on osx?

**Reply by PopeyeTheSai10r** [reply]
macOS [Automator ](https://support.apple.com/guide/automator/welcome/mac), [Shortcuts ](https://support.apple.com/guide/shortcuts-mac/welcome/mac) or [Crontab](https://www.geekbitzone.com/posts/macos/crontab/macos-schedule-tasks-with-crontab/) may help with that.

**Reply by Jimpan81** [reply]
What is the command in terminal to start grabbing without to have do git clone --depth 1 -b master https://github.com/iptv-org/epg.git every time . (Done it once )

**Reply by PopeyeTheSai10r** [reply]
> What is the command in terminal to start grabbing without to have do git clone --depth 1 -b master https://github.com/iptv-org/epg.git every time . (Done it once )

https://github.com/iptv-org/epg#:~:text=Then%20also%20through,npm%20run%20grab
